### ISSUE TITLE
Expression box for while/for test may be mislocated

### ISSUE THREAD

**Issue by FrankBau** [thonny/thonny#1134]
(Thonny 3.2.7 Win10 64 Python 3.7.6 Tk 8.6.9)
When I do nicer debugging (Ctrl+F5) and stepping (F7) through my simple gcd code
```
def gcd( a, b ):
    while a != b:
        if a>b:
            a = a-b
        else:
            b = b-a
    return a
g = gcd(44,12)
```
The evaluation overlay of `a != b` is shown in a wrong line:
![Thonny_gcd](https://user-images.githubusercontent.com/5374048/78073888-fb398080-73a1-11ea-853f-adcf4c3e0119.png)

As I want to use Thonny in my 101 prog class, thats a little bit frustrating.

**Comment by aivarannamaa** [comment]
Thank you for the report! I could reproduce this and will fix it for the next release.

I'm surprised that nobody hasn't reported it earlier.

**Comment by aivarannamaa** [comment]
Fix will be in Thonny 3.3
